main.sv-geolocation.php

```diff
@@ -14,8 +14,8 @@ public function GetEditClass()
 	public function GetSQLColumns($bFullSpec = false)
 	{
 		$aColumns = array();
-		$aColumns[$this->GetCode().'_lat'] = 'DECIMAL(8,6)';
-		$aColumns[$this->GetCode().'_lng'] = 'DECIMAL(9,6)';
+		$aColumns[$this->GetCode().'_lat'] = 'DECIMAL(10,8)';
+		$aColumns[$this->GetCode().'_lng'] = 'DECIMAL(11,8)';
```

Comment:
When `GetPrecision` is implemented, this can be changed to:
```suggestion
		$iPrecision = $this->GetPrecision();

		$aColumns = array();
		$aColumns[$this->GetCode().'_lat'] = sprintf('DECIMAL(%d,%d)', 2+$iPrecision, $iPrecision);
		$aColumns[$this->GetCode().'_lng'] = sprintf('DECIMAL(%d,%d)', 3+$iPrecision, $iPrecision;)
```